src/device-registry/bin/www

```diff
@@ -120,7 +121,9 @@ const run = async() => {
                 )}`
                     );
                 } else {
-                    const { error, value } = eventsSchema.validate(measurements);
+
+                    const cleanedMeasurements = data.map(obj => cleanDeep(obj));
```

Comment:
@BenjaminSsempala 
I have used a package called [clean-deep](https://www.npmjs.com/package/clean-deep) to remove any NaNs and etc before proceeding with input validations.

Reply:
Thanks @Baalmart ...Makes sense